cdbsearch.py

```diff
@@ -360,6 +368,24 @@ async def search(self, board, depth):
         if scored_db_moves == {}:
             return 0, ["invalid"]
 
+        # stop search if we are in EGTB and know the result
+        if (
+            not self.TBsearch
+            and sum(p in "pnbrqk" for p in board.epd().lower().split()[0]) <= CDB_EGTB
+        ):
+            bestmove, bestscore = max(
+                [t for t in scored_db_moves.items() if t[0] != "depth"],
+                key=lambda t: t[1],
+            )
+            if bestscore == 0:
+                return 0, [bestmove, "draw"]
```

Comment:
So at the moment we return "draw" for a TB draw, but "EGTB" for win/loss (user can check eval, I did not want to introduce too many terminal cases). Arguably, we could also return "EGTB" for TB draws, user can see from 0 eval that it is a draw. What do you think?

Reply:
I think we could use EGTB for both TB win and draw, since these are 'proven' results. That is different from a typical draw eval.